.npmrc

```diff
@@ -0,0 +1 @@
+legacy-peer-deps=true
```

Comment:
required because the airbnb eslint config doesn't work with the current version of eslint. old versions of npm did not install peer dependencies automatically, which is where this conflict comes from with their specified eslint version and ours (the latest).